### PR TITLE
Feature/MOBILE-3090 customer sdk project enhancements

### PR DESCRIPTION
**Pull request by theido**
https://bringg.atlassian.net/browse/MOBILE-3090

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    macos:
+      xcode: 10.2.0
```
Comment (idomizrachi-bringg) [inline .circleci/config.yml]: We should move to - 10.3.0
it is supported by circle ci

File: BringgTrackingFramework/fastlane/Appfile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# app_identifier("[[APP_IDENTIFIER]]") # The bundle identifier of your app
```
Comment (idomizrachi-bringg) [inline BringgTrackingFramework/fastlane/Appfile]: This looks like all lines are commented out
consider removing this file

File: circle.yml [hunk 3]
```diff
@@ -1,44 +0,0 @@
-general:
-
-machine:
-  xcode:
-    version: "8.3.2"
```
Comment (idomizrachi-bringg) [inline circle.yml]: 😟

**Comment by idomizrachi-bringg** [comment]
Build is failing - failure to run fastlane

File: .circleci/config.yml (hunk 1 above)
Comment (theido) [reply .circleci/config.yml]: done

File: BringgTrackingFramework/fastlane/Appfile (hunk 2 above)
Comment (theido) [reply BringgTrackingFramework/fastlane/Appfile]: done

**Pull request merged**